napari/utils/misc.py

```diff
@@ -183,23 +188,26 @@ def ensure_sequence_of_iterables(
     In [2]: ensure_sequence_of_iterables([(1, 2), (3, 4)])
     Out[2]: [(1, 2), (3, 4)]
 
-    In [3]: ensure_sequence_of_iterables({'a':1})
-    Out[3]: repeat({'a': 1})
+    In [3]: ensure_sequence_of_iterables([(1, 2), None])
+    Out[3]: [(1, 2), None]
```

Comment:
this isn't true without `allow_none=True`.

```python
In [3]: ensure_sequence_of_iterables([(1, 2), None])
Out[3]: repeat([(1, 2), None])
```

Reply:
oops, nice catch!